docs/usage/self-hosting.md

```diff
@@ -2,43 +2,45 @@
 
 ## Installing Renovate OSS CLI
 
-#### npmjs
+### npmjs
 
 ```sh
 $ npm install -g renovate
 ```
 
-Since renovate v20 `npm`, `pnpm` and `yarn` are no longer embedded, so you need to install them globally if you need to update lockfiles.
+Starting from version 20, Renovate does not embed `npm`, `pnpm` and `yarn` anymore.
+If you want to use these package managers to update your lockfiles, you must install them globally.
 
 ```sh
 $ npm install -g yarn pnpm
 ```
 
-The same goes for any other third party binary tool that may be needed, such as `gradle` or `poetry` - you need to make sure they are installed and the appropriate version you need before running Renovate.
+The same goes for any other third party binary tool like `gradle` or `poetry` - you need to make sure they are installed and the appropriate version before running Renovate.
 
-#### Docker
+### Docker
 
 Renovate is available for Docker via an automated build [renovate/renovate](https://hub.docker.com/r/renovate/renovate/).
 It builds `latest` based on the `master` branch and all semver tags are published too.
-All the following are valid:
+For example, all the following are valid tags:
 
 ```sh
 $ docker run --rm renovate/renovate
-$ docker run --rm renovate/renovate:19.181.2
-$ docker run --rm renovate/renovate:19.181
-$ docker run --rm renovate/renovate:19
+$ docker run --rm renovate/renovate:24.9.4
```

Comment:
Maybe we should configure a regex manager to update this regularly. 🤔

Reply:
That would be nice, to not have to update this manually. 👍

Reply:
@HonkingGoose checkout #8114 😏

Reply:
Oh that's cool! 😄